pyupgrade.py

```diff
@@ -58,6 +58,11 @@
         AsyncFunctionDef = ast.AsyncFunctionDef
     else:
         AsyncFunctionDef = ast.stmt
+    SyncOrAsyncFunctionDef = Union[
+        ast.AsyncFunctionDef,
+        ast.FunctionDef,
+        ast.Lambda
```

Comment:
if you upgrade `add-trailing-comma` to the latest version it will fix this

Reply:
Addressed in 7f7709432965ef668110d1fe96f932b9f02d3a88.